src/lava/magma/core/process/ports/ports.py

```diff
@@ -254,7 +254,7 @@ class OutPort(AbstractIOPort, AbstractSrcPort):
     """
```

Comment:
Regarding AbstractSrcPort:
Maybe a stupid question, but I thought RefPorts are bi-directional. Thus, why can we label them as Source Port?

Reply:
see answer above